spec/support/factory_girl.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+
+  config.before(:suite) do
+    begin
+      DatabaseCleaner.start
```

Comment:
This code is somewhat redundant with [these lines](https://github.com/dpdawson/ArborApp/blob/feature/setup-factory-girl/spec/rails_helper.rb#L67-L78) from the rails_helper - I would recommend combining them (probably moving the DatabaseCleaner calls from the rails_helper into this file)